src/components/CodeSnippet/CodeSnippet.stories.mdx

```diff
@@ -147,3 +147,77 @@ With the `options` being an array of option properties compatible with the `Sele
   </Story>
 </Canvas>
 
+Multiple dropdowns can be passed in if needed.
+
+<Canvas>
+  <Story name="Dropdowns">
+  {() => {
+    const [channel, setChannel] = useState("stable");
+    const [snap, setSnap] = useState("firefox");
+    const code = `sudo snap install ${snap} ${channel !== 'stable' ? "--"+channel : ""}`;
```

Comment:
Up to you if you take this suggestion  - ternaries get confusing really easily, it took me a (brief) moment to wrap my head around what this one was doing.
```suggestion
    const code = `sudo snap install ${snap} ${channel === 'stable' ? "" : "--"+channel}`;
```